events/views.py

```diff
@@ -46,6 +47,7 @@ def create_event(request):
         filled_form = EventForm(request.POST)
         if filled_form.is_valid():
             filled_form.instance.author = request.user # fill in author with current user
+            filled_form.instance.signed_up.add(request.user)
             filled_form.save()
```

Comment:
Save first and then add the user to the sign up. The current order caused an error saying the event needed a value for "id" before the many-many relationship could be used.

Reply:
I did not have a problem when I ran this feature

Reply:
Fixed.
This error was only when creating a new event. The author is automatically added as an attendee. The event needs to be saved (given an id number) before the attendee can be added. The .add method automatically saved the instance. No need to call .save after. Form.instance will point to the instance in the database we just saved because it is a model form (a django class where the form is attached to a model).